examples/outputs/micro_transaction.rs

```diff
@@ -67,9 +67,9 @@ async fn main() -> Result<()> {
             ))
             // If the receiver does not consume this output, we Unlock after a day to avoid
             // locking our funds forever.
-            .add_unlock_condition(UnlockCondition::Expiration(
-                ExpirationUnlockCondition::new(address, tomorrow).unwrap(),
-            ))
+            .add_unlock_condition(UnlockCondition::Expiration(ExpirationUnlockCondition::new(
+                address, tomorrow,
```

Comment:
Not your change, but we should probably at least send from address index 0 to 1, so the address in Expiration and StorageDepositReturn is not the same as in the AddressUnlockCondition